dev/defaults.yml

```diff
@@ -147,7 +147,7 @@ projects:
           - ldap.bind_password
   oc-id:
     type: rails
-    install_options: --without development test doc
+    install_options: bundle config set --local without development test doc
```

Comment:
I'm not sure this will work. The `install_options` are flags that are being passed into `bundle install`. I'm not sure what the right option is for this file.